docker/smile/docker-command-smile.sh

```diff
@@ -13,6 +13,30 @@ export HOME=/your/home/directory
 #set server info
 export SERVER=smm.server.com
 
+# set each component's tag
```

Comment:
Same as the docker-command-smile-traefik.sh version might change.
These are the latest release
```
SMILE 0.3.2
GraphQL 0.3.0
clowder_create_collection 0.1.1
clowder_create_dataset 0.1.1
clowder_create_space 0.1.1
clowder_list 0.1.1
clowder_upload_file 0.1.3
autophrase 0.1.7
classification 0.1.3
image_crawler 0.1.4
name_entity_recognition 0.1.4
network_analysis 0.1.5
preprocessing 0.1.5
sentiment_analysis 0.1.7
topic_modeling 0.1.5
```

Reply:
updated